test/test_data/test_target_validator.py

```diff
@@ -461,6 +463,8 @@ def test_targetvalidator_inversetransform():
     y_decoded = validator.inverse_transform(y)
     np.testing.assert_array_almost_equal(y, y_decoded)
 
+    assert validator.classes == []
```

Comment:
Shouldn't this be `[[0, 1], [0, 1], [0, 1], [0, 1]]`?

Reply:
For multilabel data we do not perform encoding, because sadly we need a label binarized I think. Else it would be converted (the whole [0, 1, 0] to a single number).

To go in line with the fact that there is no encoding I leave this as empty. We can however ask if the type of target is multilabel indicator, make it so that the classes are the unique elements of the target array. Does this make sense?

Reply:
Hm, I think it's fine this way.